app/assets/javascripts/service_types/IsServiceWorking.js

```diff
@@ -0,0 +1,186 @@
+import _ from 'lodash';
+import React from 'react';
+import GenericLoader from '../components/GenericLoader';
+import ExperimentalBanner from '../components/ExperimentalBanner';
+import SectionHeading from '../components/SectionHeading';
+import Button from '../components/Button';
+import {apiFetchJson} from '../helpers/apiFetchJson';
+import ProfileBarChart from '../student_profile/ProfileBarChart.js';
+import * as Routes from '../helpers/Routes';
+import moment from 'moment';
+
+const style = {
+  container: {
+    margin: 30
+  },
+  studentRow: {
+    marginBottom: 30,
+  },
+  profileBarChartStyles: {
+    title: {
+      fontSize: 20
+    },
+    container: {
+      width: '100%',
+      margin: '10px auto',
+      position: 'relative'
+    },
+    chartHeight: 250,
+  },
+  a: {
+    fontSize: 24
+  },
+  buttonStyle: {
+    float: 'left',
+    margin: 5,
+  },
+  buttonSectionStyle: {
+    width: '100%',
+    height: 50
+  }
+};
+
+class IsServiceWorking extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      serviceTypeSelected: null // options: null, 502, 503
+    };
+
+    this.fetchIsServiceWorkingData = this.fetchIsServiceWorkingData.bind(this);
+    this.renderPage = this.renderPage.bind(this);
+  }
+
+  fetchIsServiceWorkingData() {
+    const {serviceTypeSelected} = this.state;
+
+    const url = `/api/is_service_working_json/${serviceTypeSelected.id}`;
+
+    return apiFetchJson(url);
+  }
+
+  render() {
+    return (
+      <div className="IsServiceWorking">
+        <ExperimentalBanner />
+        <SectionHeading>
+          "Is this Service Working for this Student?"
+        </SectionHeading>
+        <p style={{padding: '10px 0 0 10px'}}>
+          Every student is different.
+          One student might respond well to a service, while another might respond poorly.
```

Comment:
I like this intro text, short and trying to set the tone for curiosity. 👍 

Maybe consider rephrasing "might respond poorly" - it sort of suggests this happens because of an essential quality in the student, rather than something that's in educators' control (eg, might not have been the right fit for that student, not well-matched what they need).